### PR TITLE
add centreon documentation

### PR DESCRIPTION
**Pull request by Mary-Clb**

### CODE REVIEW

**Comment by trasher** [comment]
It probably miss an main menu entry, it does not appears on build:
https://glpi-plugins--68.org.readthedocs.build/en/68/

**Comment by orthagh** [comment]
yes, please update sources/index.rst

**Comment by trasher** [comment]
ping?

File: source/centreon/index.rst [hunk 1]
```diff
@@ -0,0 +1,61 @@
+=============================
```
Comment (orthagh) [inline source/centreon/index.rst:1]:
```suggestion
```

File: source/centreon/index.rst [hunk 2]
```diff
@@ -0,0 +1,61 @@
...
+Centreon Plugin Documentation
+=============================
+
+Centreon
+========
```
Comment (orthagh) [inline source/centreon/index.rst:6]:
```suggestion
```

File: source/centreon/index.rst [hunk 3]
```diff
@@ -0,0 +1,61 @@
...
+
+* Sources link: `<https://github.com/pluginsGLPI/centreon>`_
+* Download: `<https://github.com/pluginsGLPI/centreon/releases>`_
+
+Requirements
+============
```
Comment (orthagh) [inline source/centreon/index.rst:12]:
```suggestion
Requirements
------------
```

File: source/centreon/index.rst [hunk 4]
```diff
@@ -0,0 +1,61 @@
...
+
+PHP >= 7.4.0
+GLPI >= 10.0.0
+
+Features
+========
```
Comment (orthagh) [inline source/centreon/index.rst:18]:
```suggestion
Features
--------
```

File: source/centreon/index.rst [hunk 5]
```diff
@@ -0,0 +1,61 @@
...
+You can perform 3 main actions :
+
+* Acknowledge a problem
+* Schedule a downtime
+* Trigger a host check,
+
+and you can also see the timeline of the host.
+
+Installation
+============
```
Comment (orthagh) [inline source/centreon/index.rst:33]:
```suggestion
Installation
------------
```

File: source/centreon/index.rst [hunk 6]
```diff
@@ -0,0 +1,61 @@
...
+
+* Uncompress the archive
+* Move the ``centreon`` folder to ``<GLPI_ROOT>/plugins`` directory
+* Navigate to the **Configuration > Plugins** page,
+* Install and activate the plugin.
+
+Usage
+=====
```
Comment (orthagh) [inline source/centreon/index.rst:41]:
```suggestion
Usage
-----
```

**Review by orthagh — APPROVED** [review]

**Pull request merged**